makefile

```diff
@@ -381,51 +381,49 @@ build/%_CGRA_out.raw: build/%_pnr_bitstream
-	build=../../../build;   \
-	./run.csh top_tb.cpp    \
-		$(QVSWITCH)     \
-		-gen            \
-		-config $${build}/$*_pnr_bitstream \
-		-input  $${build}/$*_input.png     \
-		-output $${build}/$*_CGRA_out.raw  \
-		-out1 s1t0 $${build}/1bit_out.raw  \
-		-delay $(DELAY)                    \
+	./run_tbg.csh $(QVSWITCH) -gen \
+		-config    $(BUILD)/$*_pnr_bitstream \
+		-input     $(BUILD)/$*_input.png     \
+		-output    $(BUILD)/$*_CGRA_out.raw  \
+		-out1 s1t0 $(BUILD)/1bit_out.raw     \
+		-delay $(DELAY) \
 		-nclocks 5M
 
     ifeq ($(ONEBIT), TRUE)
-	mv $${build}/1bit_out.raw $${build}/$*_CGRA_out.raw
+	mv $(BUILD)/1bit_out.raw $(BUILD)/$*_CGRA_out.raw
     endif
 
-else
-	cp $(VERILATOR_TOP)/sram_stub.v $(RTL_DIR)/sram_512w_16b.v  # SRAM hack
-
-	python TestBenchGenerator/generate_harness.py \
-		--pnr-io-collateral build/$*.io.json      \
-		--bitstream build/$*_pnr_bitstream        \
-		--max-clock-cycles 5000000                \
-		--quiet                                   \
-		--output-file-name harness.cpp
-
-	# Verilator wrapper that only builds if the output object is not present
-	# (override with --force-rebuild)
-	python TestBenchGenerator/verilate.py \
-		--harness harness.cpp             \
-		--verilog-directory $(RTL_DIR)    \
-		--output-directory build          \
-		--top-module-name top
-
-	make --silent -C build -j -f Vtop.mk Vtop
-
-	# HACK: Input file name to inpurt port file name, also pre-processing input
-	# file for DELAY
-	cd build; python ../TestBenchGenerator/process_input.py $*.io.json $*_input.raw $(DELAY)
-
-	cd build; ./Vtop
-
-	# HACK: Output port file name to output file name, also post-processing
-	# output file for DELAY
-	cd build; python ../TestBenchGenerator/process_output.py $*.io.json $*_CGRA_out.raw $* $(DELAY)
 endif
+# else
+# 	cp $(VERILATOR_TOP)/sram_stub.v $(RTL_DIR)/sram_512w_16b.v  # SRAM hack
+# 
+# 	python TestBenchGenerator/generate_harness.py \
+# 		--pnr-io-collateral build/$*.io.json      \
+# 		--bitstream build/$*_pnr_bitstream        \
+# 		--max-clock-cycles 5000000                \
+# 		--quiet                                   \
+# 		--output-file-name harness.cpp
+# 
+# 	# Verilator wrapper that only builds if the output object is not present
+# 	# (override with --force-rebuild)
+# 	python TestBenchGenerator/verilate.py \
+# 		--harness harness.cpp             \
+# 		--verilog-directory $(RTL_DIR)    \
+# 		--output-directory build          \
+# 		--top-module-name top
+# 
+# 	make --silent -C build -j -f Vtop.mk Vtop
+# 
+# 	# HACK: Input file name to inpurt port file name, also pre-processing input
+# 	# file for DELAY
+# 	cd build; python ../TestBenchGenerator/process_input.py $*.io.json $*_input.raw $(DELAY)
+# 
+# 	cd build; ./Vtop
+# 
+# 	# HACK: Output port file name to output file name, also post-processing
+# 	# output file for DELAY
+# 	cd build; python ../TestBenchGenerator/process_output.py $*.io.json $*_CGRA_out.raw $* $(DELAY)
+# endif
```

Comment:
Can we just remove this if we're no longer going to use it? We can always search in the commit history to find the old code if we want to revive it.

Reply:
yes of course...i will take care of it.
meanwhile it looks like checks are failing now, so i will look into that as well